src/API/Plugin.php

```diff
@@ -19,6 +21,14 @@ class Plugin extends Client
     const SETTING_EDITABLE_KEY = 'editable';
     const SETTING_MODIFIED_DATE_KEY = 'modified_on';
 
+    /**
+     * @param IntegrationInterface $integration
+     */
+    public function __construct(IntegrationInterface $integration)
+    {
+        parent::__construct($integration);
+    }
```

Comment:
Apparently its not.  Magento was giving me errors saying `__construct()` not found on `Client` and `Plugin`.